### ISSUE TITLE
Latest release (c3@0.4.11-rc4) still using "d3": "<=3.5.0" in its dependencies

### ISSUE THREAD

**Issue by wuct** [c3js/c3#1637]
The c3@0.4.11-rc4 on the official npm registry still using `"d3": "<=3.5.0"` in its dependencies. Please release a new version that use `"d3": "~3.5.0"`.
# To reproduce

```
npm i c3@0.4.11-rc4
cat node_modules/c3/package.json | grep d3 // print d3": "<=3.5.0"
```

**Comment by aendrew** [comment]
@masayuki0812 Thoughts? Might be worth releasing a really quick RC5 with this change and maybe everything else that's in the `dev` branch.

**Comment by wiibaa** [comment]
@aendrew this is fixed with 0.4.11 release, right ?

**Comment by aendrew** [comment]
Yep! Thanks @wiibaa!
